kopf/config.py

```diff
@@ -7,7 +7,6 @@
 LOGLEVEL_ERROR = logging.ERROR
 LOGLEVEL_CRITICAL = logging.CRITICAL
 
-
```

Comment:
As far as I remember the styling guides and linters, there should be 2 empty lines before functions and classes in the module (but 1 empty line before constants and values of different kinds, including types (unless they use `class`)).

It would not be a blocker for this PR per se. But since my action next time I revisit this file would be to re-add the line, I would like to clarify the reason for this removal: was it intentional? Or just accidentally happened, and it is fine to re-add it?

Reply:
In case of formatting I would love to propose using black. Then there's no need to worry about anything. But that's something out of the scope of this PR as it will introduce a lot of changes and should be discussed before.